### ISSUE TITLE
Minor curiosity: why are some scorings (e.g MoNA) not normalized?

### ISSUE THREAD

**Issue by rickhelmus** [ipb-halle/MetFragRelaunched#24]
When trying to reconstruct relative scores from the absolute values returned by MetFrag CL I noticed that some scorings (e.g. MoNA) are not normalized. I am just curious why this is.

Thanks,
Rick

**Comment by schymane** [comment]
All scores should be normalized when the total score is calculated (between 0 and 1, then adjusted with the weighing given), but in the results file the raw scores are exported, as these are also useful (and it’s easy to repeat the normalization). MoNA Exact Similarity should be between 0 and 1, whereas the MoNA MetFusion-style score can be greater than one – this is a function of the way it is calculated (see e.g. the MetFusion paper). If the MetFusion score is far above one it’s a sign that there are a lot of similar spectrum/structure combinations adding weight to the identification, if they are well below one there’s not much similar on record … does that help?

**Comment by rickhelmus** [comment]
Hi Emma,

Thanks for your explanation! This is quite interesting, I will need to have a better look at the MetFusion paper though. For now if I understand correctly what is desired here is to keep the original scaling in order to judge what is going on. I do wonder know though if it is 'fair' that >1 MetFusion scores should be able to influence the final score with a greater weight since they are not normalized? Perhaps it would make more sense to report the non-normalized scores but normalize them before calculating the total score?

**Comment by schymane** [comment]
The MetFusion scores are normalized along with everything else, so they only count a maximum of 1 to the total score in the end. The values that are greater than one that you see are the non-normalised “raw” values. (c.f. publications, also normalized to 1 in the final score, but sometimes in the 100s or 1000s in the “raw” values)

**Comment by rickhelmus** [comment]
Hi Emma,

Thanks again. It seems I got confused and mixed the similarity and exact scores. You're completely right that the similarity is normalized.

The whole reason I started this discussion was an example like this:

![image](https://user-images.githubusercontent.com/37015911/52955580-4e629680-338d-11e9-93de-036eb2f8a582.png)


Normalizing this value would yield a value of 1.0 (since it is the only result), but as you mention since exact scores are already scaled 0-1.0, this doesn't seem useful. That clears it up for me :-)
